challenges/02-javascript-algorithms-and-data-structures/functional-programming.json

```diff
@@ -655,7 +655,7 @@
         "Another useful array function is <code>Array.prototype.filter()</code>, or simply <code>filter()</code>. The <code>filter</code> method returns a new array which is at most as long as the original array, but usually has fewer items.",
         "<code>Filter</code> doesn't alter the original array, just like <code>map</code>. It takes a callback function that applies the logic inside the callback on each element of the array. If an element returns true based on the criteria in the callback function, then it is included in the new array.",
         "<hr>",
-        "The variable <code>watchList</code> holds an array of objects with information on several movies. Use a combination of <code>filter</code> and <code>map</code> to return a new array of objects with only <code>title</code> and <code>rating</code> keys, but where <code>imdbRating</code> is greater than or equal to 8.0. Note that the rating values are saved as strings in the object and you may want to convert them into numbers to perform mathematical operations on them."
+        "The variable <code>watchList</code> holds an array of objects with information on several movies. Use a combination of <code>filter</code> and <code>map</code> to return a new array of objects with only <code>Title</code> and <code>imdbRating</code> keys, but where <code>imdbRating</code> is greater than or equal to 8.0. Note that the rating values are saved as strings in the object and you may want to convert them into numbers to perform mathematical operations on them."
```

Comment:
"The variable <code>watchList</code> holds an array of objects with information on several movies. Use <code>filter</code> to create a filtered array that includes all movies with an <code>imdbRating</code> greater than or equal to 8.0. Then, use <code>map</code> to return a new array of objects with only the <code>Title</code> and <code>imdbRating</code> keys as <code>title</code> and <code>rating</code>, respectively. Note that the rating values are saved as strings in the object and you may want to convert them into numbers to perform mathematical operations on them."